src/Pages/Cart/Cart.css

```diff
@@ -0,0 +1,139 @@
+.empty-cart-message-container
+{
+    margin-top: 2rem;
+    min-width: 31rem;
+}
+
+.cart-content-container
+{
+    display: flex; 
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    /* min-height: 100vh; */
+    padding: 3rem;
+}
+
+.cart-grid
+{
+    display: grid;
+    width: 80%;
+    min-width: fit-content;
+    padding: 2rem;
+    gap: 2rem;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
repeat can be used here instead of repeating 1fr 4 times

Reply:
Done.